ci-scripts/yocto2.groovy

```diff
@@ -22,6 +22,13 @@ void setupBitbake(String yoctoDir, String templateConf, boolean doArchiveCache,
 
         // Add other settings that are CI specific to the local.conf
         sh "cat /workspace/conf/local.conf.appendix >> ${yoctoDir}/build/conf/local.conf"
+
+        // Uncomment ACCEPT_FSL_EULA = "1" to accept the Freescale EULA
+        // if ACCEPT_FSL_EULA parameter is set to true
+        if (getBoolEnvVar(freescaleEULA) == true) {
```

Comment:
String and default value as arguments. (And remove "== true", I think.)

Reply:
rephrase, please